### PR TITLE
review placeholder issue

### PR DESCRIPTION
**Pull request by vairafiq**

### CODE REVIEW

**Review by obiPlabon — CHANGES_REQUESTED** [review]
There's best practice for such cases. Maybe you can try that. Here's an example...
```php
<?php echo esc_attr( sprintf( _x( 'Select %s', 'Rating search select placeholder', 'directorist' ), $data['label'] ) ); ?>
```

**Review by obiPlabon — DISMISSED** [review]
Code reviewed

File: templates/search-form/fields/review.php [hunk 1]
```diff
@@ -13,10 +13,11 @@
 		<label><?php echo esc_html( $data['label'] ); ?></label>
 	<?php endif; ?>
 	<div class="directorist-select" id="directorist-review-select-js">
-		<select name='search_by_rating' <?php echo ! empty( $data['required'] ) ? 'required="required"' : ''; ?> data-isSearch="true">
+		<select name='search_by_rating' <?php echo ! empty( $data['required'] ) ? 'required="required"' : ''; ?> data-placeholder="<?php echo esc_attr( sprintf( _x( 'Select %s', 'Rating search select placeholder', 'directorist' ), $data['label'] ) ); ?>
```
Comment (kowsar89) [inline templates/search-form/fields/review.php:16]: Please keep it in single line like before

**Review by kowsar89 — CHANGES_REQUESTED** [review]

**Review by kowsar89 — APPROVED** [review]
Looks good

**Review by obiPlabon — APPROVED** [review]
Okay

**Pull request merged**